src/docs/cookbook-payments/removing-stripe/index.md

```diff
@@ -0,0 +1,332 @@
+transaction process, email templates, checkout page) more depending on your
+marketplace idea. You might also want to remove e.g. unused Stripe components
+from your project to clean up the code.
+
+###1. Update the transaction process and remove Stripe related actions
+
+You need to remove at least the following actions:
+
+- :action/stripe-create-payment-intent
+- :action/stripe-capture-payment-intent
+- :action/stripe-confirm-payment-intent
+- :action/stripe-create-payout
+- :action/stripe-refund-charge
+- :action/stripe-refund-payment
+
+If you are using the FTW default process with strong customer authentication
+(e.g. flex-default-process) you need to remove confirming the payment.
+Otherwise, the transaction will get stuck. The simplest way to do this is to
+remove `pending-payment` and `expire-payment` states and point `request-payment`
+and `equest-payment-after-enquiry` directly to `preauthorized` state. However,
+when you are building the transaction process to your marketplace it might make
+sense to rename some of the states and transitions (e.g. preauthorized) so that
+they make more sense in your use-case.
+
+#### Edit util/transaction.js to reflect the changes in your transaction process
+
+In
+[src/util/transaction.js](https://github.com/sharetribe/flex-template-web/blob/master/src/util/transaction.js)
+file, there are lots of helper functions that are used to determine which state
+the transaction is. This file should be updated to match the new transaction
+process.
+[Read more about editing transaction.js](/cookbook-transaction-process/change-transaction-process-in-ftw/#2-check-if-the-transactionjs-file-needs-to-be-updated)
+
+Example:
+
+```diff
+     [STATE_INITIAL]: {
+       on: {
+         [TRANSITION_ENQUIRE]: STATE_ENQUIRY,
+-        [TRANSITION_REQUEST_PAYMENT]: STATE_PENDING_PAYMENT,
++        [TRANSITION_REQUEST_PAYMENT]: STATE_PREAUTHORIZED,
+       },
+     },
+     [STATE_ENQUIRY]: {
+       on: {
+-        [TRANSITION_REQUEST_PAYMENT_AFTER_ENQUIRY]: STATE_PENDING_PAYMENT,
++        [TRANSITION_REQUEST_PAYMENT_AFTER_ENQUIRY]: STATE_PREAUTHORIZED,
+       },
+     },
+-
+-    [STATE_PENDING_PAYMENT]: {
+-      on: {
+-        [TRANSITION_EXPIRE_PAYMENT]: STATE_PAYMENT_EXPIRED,
+-        [TRANSITION_CONFIRM_PAYMENT]: STATE_PREAUTHORIZED,
+-      },
+-    },
+-
+-    [STATE_PAYMENT_EXPIRED]: {},
+     [STATE_PREAUTHORIZED]: {
+       on: {
+         [TRANSITION_DECLINE]: STATE_DECLINED,
+
+```
+
+###2. Remove Stripe checks from EditListingWizard
+
+In
+[`EditListingWizard.js`](https://github.com/sharetribe/ftw-daily/blob/master/src/components/EditListingWizard/EditListingWizard.js)
+we are checking if the provider has a Stripe account with all the required
+information before we allow them to publish listings. This check is done in the
+`handlePublishListing` function and needs to be removed.
+
+```js
+  handlePublishListing(id) {
+    const { onPublishListingDraft} = this.props;
+    onPublishListingDraft(id);
```

Comment:
```suggestion
    this.props.onPublishListingDraft(id);
```

And actually, this whole function could be removed, but that would require another line to be edited (the one where this.handlePublishListing(id) is called. For the sake of this guide, it's probably better to keep the change inside this one method.